README.md

```diff
@@ -88,10 +88,14 @@ go run _examples/booktown-books/main.go
 
 ## Changelog
```

Comment:
I think you should just note in the README the versions released, and link to a new CHANGELOG.md file, that is where you should include all release notes otherwise this readme is going to get very noisy